install/linux/docker-ce/centos.md

```diff
@@ -131,56 +131,51 @@ from the repository.
 
 #### Install Docker CE
 
-1.  Install the latest version of Docker CE, or go to the next step to install a
-    specific version.
+1.  Install the _latest version_ of Docker CE, or go to the next step to install a specific version:
 
     ```bash
     $ sudo yum install docker-ce
     ```
 
-    > **Warning**: If you have multiple Docker repositories enabled, installing
+    If prompted to accept the GPG key, verify that the fingerprint matches
+    `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35`, and if so, accept it.
+
+    > Got multiple Docker repositories?
+    >
+    > If you have multiple Docker repositories enabled, installing
     > or updating without specifying a version in the `yum install` or
     > `yum update` command always installs the highest possible version,
     > which may not be appropriate for your stability needs.
-    {:.warning}
+    {:.warning-vanilla}
```

Comment:
This notation applies a CSS class, so you're basically creating

```
<blockquote class="warning-vanilla">
```

But we don't have any CSS declaration for that class. If you want this note to render in blue, just delete the `{...}`

I see that this class is also being used in other places, so consider doing a search/replace.